tests/next_tests/integration_tests/feature_tests/ffront_tests/test_gt4py_builtins.py

```diff
@@ -137,10 +138,7 @@ def conditional_nested_tuple(
         return where(mask, ((a, b), (b, a)), ((5.0, 7.0), (7.0, 5.0)))
 
     size = cartesian_case.default_sizes[IDim]
-    bool_field = np.random.choice(a=[False, True], size=(size))
-    mask = cases.allocate(cartesian_case, conditional_nested_tuple, "mask").strategy(
-        cases.ConstInitializer(bool_field)
-    )()
+    mask = cartesian_case.as_field([IDim], np.random.choice(a=[False, True], size=size))
```

Comment:
could we built in to cases that boolean fields are allocated with alternating values by default?

Reply:
maybe in a follow-up PR to get the GPU support in